action.yml

```diff
@@ -40,9 +40,13 @@ inputs:
   GITHUB_TOKEN:
     description: 'Github token of the repository (automatically created by Github)'
     required: true
+  use_bandit_yaml:
+    description: 'set to true if bandit.yaml file is used. expected to be at the root of the path variable'
+    required: false
+    default: 'false'
 runs:
   using: 'docker'
-  image: 'docker://epsylabs/action-bandit'
+  image: 'docker://mglivn/action-bandit:DEVOP-13'
```

Comment:
we need to rebuild the docker image because we are changing the entrypoint.sh. I plan to upload the docker image with changes to Dockerhub as 'docker://epsylabs/action-bandit' (instead of 'docker://mglivn/action-bandit:DEVOP-13'). I will do it as soon as I get access to epsylabs docker hub.